src/twilio/twilio_utils.bal

```diff
@@ -84,3 +81,14 @@ function convertToBoolean(json|error value) returns boolean {
     }
     return false;
 }
+
+function prepareError(string message, error? err = ()) returns Error {
+    log:printError(message, err);
```

Comment:
Do we need to log the error here?

Reply:
Yes. That's not needed. Removed with https://github.com/ballerina-platform/module-ballerinax-twilio/pull/64/commits/c84729156582e6ec0d1b310fa6ccf11f105b9a6b